pootle/apps/pootle_store/models.py

```diff
@@ -752,6 +752,14 @@ def istranslated(self):
     def get_suggestions(self):
         return self.suggestion_set.pending().select_related('user').all()
 
+    def get_latest_target_submission(self):
+        submission = (self.submission_set.select_related('suggestion__user')
+                      .filter(field=SubmissionFields.TARGET)
+                      .order_by('-creation_time', '-id')
+                      .first())
+        if submission:
+            return submission
```

Comment:
I am wondering if it is ok to just return the submission here. `first()` should return the submission or `None`.